specification/hugr.md

```diff
@@ -719,7 +719,7 @@ resources:
   types:
   - name: QubitVector
     # Opaque types can take type arguments, with specified names
-    args: [["size", u64]]
+    args: [["size", Int]]
```

Comment:
I changed all the `u64`s to `Int` to reflect what's in `ClassicType` and `TypeParam` - and the general policy that the difference between signed and unsigned is in the operations, not the bits flowing around the wires of the Hugr.

However, there is an argument that the specification here is not just about the (signedness-independent) serialized form of the parameters, but also an instruction as to how int constants should be parsed / treated by the frontend. If this is left as Int, then *some other* data somewhere needs to tell any frontend that the value should be unsigned...

Reply:
Are there any cases where we actually want signed semantics in the uses of `Int` in the core, or should we just say that it's always unsigned (and maybe rename to `Nat` or something)?

Reply:
I admit I haven't seen a use for a signed int in a type yet.